packages/snaps-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.3.0]
+
+### Changed
+
+- Bump `vite` from `6.2.3` to `6.2.6` ([#3305](https://github.com/MetaMask/snaps/pull/3305)), ([#3321](https://github.com/MetaMask/snaps/pull/3321))
```

Comment:
This does not affect production dependencies so it doesn't need to be mentioned.